canvas/canvas.txt

```diff
@@ -0,0 +1,297 @@
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+<canvasint> = <interestingint>
+<canvasint> = <fuzzint>
+<canvasint> = <largeint>
+
+<largeint> = 536870911
+<largeint> = 536870912
+<largeint> = 1073741823
+<largeint> = 1073741824
+<largeint> = 2147483647
+<largeint> = 2147483648
+<largeint> = 4294967295
+<largeint> = 4294967296
+
+<fuzzstring> = <fuzzstringpart>
+<fuzzstring> = <fuzzstringpart> + <fuzzstringpart>
+<fuzzstring> = <fuzzstringpart> + <fuzzstringpart> + <fuzzstringpart>
+<fuzzstringpart> = "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+<repeatcount> = 17
+<repeatcount> = 65
+<repeatcount> = 257
+<repeatcount> = 1025
+<repeatcount> = 4097
+<repeatcount> = 65537
+<repeatstr> = String.fromCharCode(<int min=0 max=127>)
+<repeatstr> = String.fromCharCode(<int min=0 max=127>).repeat(<repeatcount>)
+
+<fillRule> = "nonzero"
+<fillRule> = "evenodd"
+
+<lineCap> = "butt"
+<lineCap> = "round"
+<lineCap> = "square"
+<lineJoin> = "bevel"
+<lineJoin> = "round"
+<lineJoin> = "miter"
+
+<textAlign> = "left"
+<textAlign> = "right"
+<textAlign> = "center"
+<textAlign> = "start"
+<textAlign> = "end"
+
+<textBaseline> = "top" 
+<textBaseline> = "hanging"
+<textBaseline> = "middle"
+<textBaseline> = "alphabetic"
+<textBaseline> = "ideographic"
+<textBaseline> = "bottom"
+
+<txtDirection> = "ltr"
+<txtDirection> = "rtl"
+<txtDirection> = "inherit"
+
+<repetition> = "repeat" 
+<repetition> = "repeat-x" 
+<repetition> = "repeat-y" 
+<repetition> = "no-repeat"
+
+<imageFormat> = "image/png"
+<imageFormat> = "image/webp"
+
+<mathOperation> = +
+<mathOperation> = -
+<mathOperation> = /
+<mathOperation> = *
+
+<globalCompositeOperation> = "source-over"
+<globalCompositeOperation> = "source-in"
+<globalCompositeOperation> = "source-out"
+<globalCompositeOperation> = "source-atop"
+<globalCompositeOperation> = "destination-over"
+<globalCompositeOperation> = "destination-in"
+<globalCompositeOperation> = "destination-out"
+<globalCompositeOperation> = "destination-atop"
+<globalCompositeOperation> = "lighter"
+<globalCompositeOperation> = "copy"
+<globalCompositeOperation> = "xor"
+<globalCompositeOperation> = "multiply"
+<globalCompositeOperation> = "screen"
+<globalCompositeOperation> = "overlay"
+<globalCompositeOperation> = "darken"
+<globalCompositeOperation> = "lighten"
+<globalCompositeOperation> = "color-dodge"
+<globalCompositeOperation> = "hard-light"
+<globalCompositeOperation> = "soft-light"
+<globalCompositeOperation> = "difference"
+<globalCompositeOperation> = "exclusion"
+<globalCompositeOperation> = "hue"
+<globalCompositeOperation> = "saturation"
+<globalCompositeOperation> = "color"
+<globalCompositeOperation> = "luminosity"
+
+<this> = this
+
+<root root=true> = <lines count=50>
+
+!lineguard try { <line>; } catch(e) { console.log(e.message) }

```

Comment:
Remove the semicolon here. Since lines already include semicolons, this results in every line ending with ;;

Reply:
Fixed that, now should be good!